topfreegames/pitaya
# Message package be public

**Description**

Found out that the message package is private, which makes lots of other projects that use it unable to  access it properly. It creates difficulties such as knowing the message type, how to handle it and so on. One example is the pitaya bot project.

**How to Test it**

See if everything continues working correctly and see if it is viable to merge what I am proposing. If not, why.
